src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -163,6 +170,16 @@ public static int Main(string[] args)
                 createAnalyzerRulesMissingDocumentationFile();
             }
 
+            if (fileNamesWithValidationFailures.Count > 0)
+            {
+                Console.WriteLine("Found validation errors the following file(s). Consider re-generating the files using `MSBuild /t:pack` command:");
```

Comment:
I think we should give a more useful and actionable error message.
```suggestion
                Console.WriteLine("One or more auto-generated files were edited manually. Please revert manual changes to these files and then run `msbuild /t:pack` for each solution at the root of the repo to automatically update them.");
```